libbeat/logp/logp.go

```diff
@@ -21,8 +24,18 @@ type Logging struct {
 	ToSyslog  *bool `config:"to_syslog"`
 	ToFiles   *bool `config:"to_files"`
 	Level     string
+	Metrics   LoggingMetricsConfig `config:"metrics"`
```

Comment:
Now I see it on line 170. Would probably make sense to also use a ucfg defaultConfig.